package.json

```diff
@@ -44,5 +45,10 @@
     "ava": "*",
     "tempfile": "^2.0.0",
     "xo": "*"
-  }
+  },
+  "xo": {
+		"ignores": [
```

Comment:
The indentation is off. These lines are using tabs and the others are using spaces.

I know @sindresorhus switched to tabs for package.json because npm preserves them nowadays. But think it should be done in a separate commit.

Reply:
👍